react/__tests__/components/Logo.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import Logo from '../../Logo'
+import { render } from 'enzyme'
+
+describe('<Logo /> component', () => {
+  const renderComponent = customProps => {
+    const props = {
+      ...customProps,
+    }
+    const comp = <Logo {...props} />
```

Comment:
```suggestion
    const comp = <Logo {...customProps} />
```
Just delete the `props` constant and use customProps.